### ISSUE TITLE
page title needs to change when the timetree changes

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/lenape-timetree#263]
accessibility review feedback from @itmaybejj:

> All pages have the same title; it is very hard from a screen reader to tell when a filter is active. The information is somewhat prominent visually, but buried in the content tree.

suggestion:
> E.g., after clicking through from the tag index to a filtered page, the document title should probably become "Dutch settlers, Lunaapahkiing Princeton Timetree"

## testing notes
- [x] document title in your browser should update to include the tag when you select a tag from leaf details
- [x] after selecting a tag, when you clear the active tag your document title should be updated with the tag name removed
- [x] when you select a tag from the tag index page, the document title should include the tag name

**Comment by mnaydan** [comment]
Also looks good after following all the testing instructions. Browser tab title updates and clears as expected from home page and tag index page.
